.github/workflows/build.yaml

```diff
@@ -53,11 +53,15 @@ jobs:
                   queries: +security-and-quality
 
             - name: Bootstrap
-              run: scripts/build/gn_bootstrap.sh
+              run: |
+                  apt-get update && apt-get install -y libdmalloc-dev;
```

Comment:
I think this should be placed in the CHIP dockerfile instead

Reply:
I agree; I am just averse to touching Docker.

Reply:
100% to this

Reply:
Linux build bot is expected to fail until the updated docker image is available.